TTS/utils/audio.py

```diff
@@ -429,3 +428,13 @@ def quantize(x, bits):
     @staticmethod
     def dequantize(x, bits):
         return 2 * x / (2 ** bits - 1) - 1
+
+def _log(x, base):
+    if base == 10:
```

Comment:
I guess you were planing to raise a ```ValueError``` if the base is undefined.

Reply:
Oh I think you misunderstood me when you said:
> fix looks cool!

The code I shared wasn't a fix, it's already in the init. That's why it's already impossible to have a base other than 10 or np.e in the functions I added without triggering an exception. Do you still want me to add another `raise ValueError` ?

Reply:
I see.... Then it is my bad. Thx ! Time to merge.